spec/integration/post_chat_quote_spec.rb

```diff
@@ -139,4 +139,65 @@
       </div>
     COOKED
   end
+
+  it "correctly renders inline and non-inline oneboxes combined with chat quotes" do
```

Comment:
does it need such a complex onebox to demonstrate the issue?

Reply:
if possible I would write the minimum length possible code that would reproduce. Less to maintain in the future.

Reply:
Yes true, I ripped out the extra data from the example onebox, I had just copied it from the problem case internally.